### PR TITLE
Add UUID access, echo cmd

### PR DESCRIPTION
**Pull request by Wetmelon**
Also makes the console a bit easier to read when handling a bunch of messages.

Addresses #54

### CODE REVIEW

File: Firmware/MotorControl/utils.h [hunk 1]
```diff
@@ -13,11 +75,11 @@
 // as per the magnitude invariant clarke transform
 // The magnitude of the alpha-beta vector may not be larger than sqrt(3)/2
 // Returns 0 on success, and -1 if the input was out of range
-int SVM(float alpha, float beta, float* tA, float* tB, float* tC);
+int SVM(float alpha, float beta, float *tA, float *tB, float *tC);
```
Comment (madcowswe) [inline Firmware/MotorControl/utils.h]: why did you change this?

Comment (Wetmelon) [reply Firmware/MotorControl/utils.h]: I auto-formatted the document to clean up the defines and I guess it didn't like where the *'s were.
